### PR TITLE
Release v15.0.0

### PR DESCRIPTION
**Pull request by dvdplm**
Release prep for v14.3.0﻿

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```
Comment (bkchr) [inline Cargo.toml:2]: If you bump to `2021` edition this should be done automatically.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,29 +1,45 @@
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [14.3.0] - 2022-02-08
+
+### Changed
+
+- Update edition to 2021
+- Update `scale-info` to v2.0.0 and `parity-scale-codec` to v3.0.0 [#35](https://github.com/paritytech/frame-metadata/pull/30)
```
Comment (ordian) [inline CHANGELOG.md:16]: This is a breaking change, unless you do `>=x, <y` trick
```suggestion
- Update `scale-info` to v2.0.0 and `parity-scale-codec` to v3.0.0 [#35](https://github.com/paritytech/frame-metadata/pull/35)
```

Comment (dvdplm) [reply CHANGELOG.md]: Fair. Arguably switching edition is also breaking (MSRV). v15 it is.

**Review by ordian — APPROVED** [review]

**Review by wigy-opensource-developer — APPROVED** [review]
LGTM. `rust-version = 1.56.1` was added to most `Cargo.toml`s that has `edition = 2021`, but because of transitive dependencies it is not compulsory.

**Pull request merged**